layouts/shortcodes/docs/instrumentation/exporters-intro.md

```diff
@@ -0,0 +1,29 @@
+<!-- cspell:ignore isset cond -->
+
+{{ $lang := .Get 0 -}} {{ $data := index $.Site.Data.instrumentation $lang }}
+{{ $name := cond (isset $.Site.Data.instrumentation $lang) (printf "OpenTelemetry %s" $data.name) "the language specific implementations" -}}
+
+In order to visualize and analyze your telemetry, you will need to export your
+data to an [OpenTelemetry Collector](/docs/collector/) or a backend such as
+[Jaeger](https://jaegertracing.io/), [Zipkin](https://zipkin.io/),
+[Prometheus](https://prometheus.io/) or a [vendor-specific](/ecosystem/vendors/)
+one.
+
+As part of {{ $name }} you will find many exporters being available. Among them,
+the OpenTelemetry Protocol (OTLP) exporters provide the best experience for you
+as an end-user:
+
+- it is suitable to be used between instrumented applications, collectors and
+  backends.
+- it has high reliability, low CPU usage and imposes minimal pressure on memory
```

Comment:
I see that these are [design goals](https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/design-goals.md#design-goals-for-opentelemetry-wire-protocol). Don't think we should state "it has high reliability, etc." here. Reliability/perf etc. depends on individual implementations.

Also, the fact that it can be used between collector and backend - Is that relevant for an application owner?

Reply:
> I see that these are [design goals](https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/design-goals.md#design-goals-for-opentelemetry-wire-protocol). Don't think we should state "it has high reliability, etc." here. Reliability/perf etc. depends on individual implementations.

fair point, let me drop them or reword that a little bit

> Also, the fact that it can be used between collector and backend - Is that relevant for an application owner?

maybe? Depends on how involved they are with the people responsible for the backend. also the docs are not only for application owners, so the list _may_ contain points not relevant for the individual reader.

Reply:
@cijothomas I removed that list for now and only put a _"the best experience for you
as an end-user, since it is a general-purpose telemetry data delivery protocol
designed in the scope of the OpenTelemetry project."_

This is far from perfect, but better than the current wording which only says "it's best for you", hope that works for you

Reply:
Looks good. If there is a refactor in the future, consider adding something along the lines of:

1. OTLP Exporters are built to carry OTel data model without any loss of information. (Other exporters may have info loss, like some exporters may not have a good place to store resources, instrumentation scope etc. but OTLP carry everything).
2. OTLP is the only exporter that supports all 3 signals (logs/metrics/traces).